resources/asciidoctor/lib/elastic_include_tagged/extension.rb

```diff
@@ -25,8 +25,8 @@ def process(_doc, reader, target, attrs)
       return target
     end
     tag = attrs[1]
-    start_match = /^(\s*).+tag::#{tag}\b/
-    end_match = /end::#{tag}\b/
+    start_match = /^(\s*).+tag::#{tag}\b/.freeze
+    end_match = /end::#{tag}\b/.freeze
```

Comment:
Is there a specific reason why you are freezing the Regexp's?
I usually freeze Strings when I use them as constants, as they are mutable. Things like Regexp's, that are immutable, are probably not necessary to freeze.

Reply:
It [*looks*](https://github.com/rubocop-hq/rubocop/pull/6333) like regexes are *technically* mutable even if they don't have any mutation methods. Weird. I suspect some of this weirdness is how I got here. I'll stop freezing them.